examples/02-plot/anti-aliasing.py

```diff
@@ -0,0 +1,95 @@
+"""
+.. _antialiasing_example:
+
+Anti-Aliasing
+~~~~~~~~~~~~~
+
+PyVista supports three types of anti-aliasing:
+
+* ``"ssaa"`` - Super-Sample Anti-Aliasing
+* ``"mxaa"`` - Multi-Sample Anti-Aliasing
+* ``"fxaa"`` - Fast Approximate Anti-Aliasing
+
+By default, anti-aliasing is disabled, but can be enabled globally with:
+
+.. code:: python
+
+   >>> import pyvista as pv
+   >>> pv.global_theme.antialiasing = 'ssaa'
```

Comment:
I'd love to, but it's actually harder than it looks.
- `antialiasing` as a property with a setter won't work for the `Plotter` as there are various configuration parameters that we'd have to pass (e.g. `all_renderers`, `multi_samples`).
- `enable_anti_aliasing` is a method, and our class is setup with just attributes. Seems weird to have an all attribute class (similar to [dataclasses](https://docs.python.org/3/library/dataclasses.html)) with a method that changes its state.

Reply:
To be clear I specifically meant `antialiasing` vs `anti_aliasing`, the underscore. So `enable_antialiasing()` rather than trying to make this a property.

Reply:
Oh! Ok, fully agree there. We can depreciate this.

Reply:
For future consideration: `pyvista/core/dataset.py` vs `pyvista/core/filters/data_set.py` :smile:

Reply:
Done in ee41207f500145104ddae02c1821ffe44159ee42.